docs/common-issues/clipped-text.md

```diff
@@ -0,0 +1,49 @@
+# Issue: Text is clipped [Draft]
+
+Long string of text not breaking to accommodate the width of the parent container. The text gets clipped.
+
+## To resolve
+
+Most recent govuk-frontend should containe fix for this issue panel heading
+https://github.com/alphagov/govuk-frontend/pull/2347.
+Or apply `.text-overflow` class on element
+This is create difficulties for people who rely on text to be resized without loss of the content.
```

Comment:
Move this line to the issue description above. At the moment it is suggesting the fix is causing them problems.